pkg/types/operation.go

```diff
@@ -89,9 +89,15 @@ type Operation struct {
 }
 
 type OperationContext struct {
-	Async             bool   `json:"async"`
-	ServicePlanID     string `json:"service_plan_id"`
-	ServiceInstanceID string `json:"service_instance_id"`
+	Async                  bool   `json:"async"`
+	IsAsyncDefinedByClient bool   `json:"by_broker_response,omitempty"`
```

Comment:
change the json property to is_async...

Reply:
it will require a migration we already have these changes persisted..